oeps/processes/oep-0021-proc-deprecation.rst

```diff
@@ -72,7 +72,15 @@ The following sections describe each state in the deprecation and removal proces
 Draft
 -----
 
-Initial state where code is identified for removal but no one has committed to coordinating the process. A ticket created with the "Deprecation (DEPR) Ticket" template remains in Draft until someone claims it as coordinator.
+Initial state where code is identified for removal but no one has committed to coordinating the process.
+
+A GitHub ticket is created with the "Deprecation (DEPR) Ticket" template in the relevant repository (either in the repository to be deprecated, or the repository where the to-be-deprecated component exists):
```

Comment:
The README on https://github.com/orgs/openedx/projects/9/views/4 has additional notes about when to use `public-engineering`.

Reply:
This comment is not a blocker, but I just meant that this section could be enhanced with some additional details.